src/cassandra/mongoose_cqerl_helper.erl

```diff
@@ -0,0 +1,12 @@
+-module(mongoose_cqerl_helper).
+-export([cqerl_client_to_pid/1]).
+
+%% Violates cqerl:client() opaqueness
+-spec cqerl_client_to_pid(cqerl:client()) -> pid().
+cqerl_client_to_pid(Client) ->
+    {Pid, _Ref} = identity(Client),
+    Pid.
+
+%% Used to supress opaque check of dialyzer
+-spec identity(any()) -> any().
+identity(X) -> X.
```

Comment:
I don't really understand why we need it. please, illustrate it on some example?

Reply:
without it you will get error that opaque type is used in mim code, while tuple is expected.